### DISCUSSION TITLE
Support showing a button's label when hovering over

### DISCUSSION THREAD

**Discussion by Amazingkenneth** [hecrj/iced#1813] [Ideas]
Like the screenshot below, many apps have the ability to display the button label when hovering over the button. Is there currently a way to implement it, or is the functionality under the roadmap?

![Hovering where the orange arrow points](https://user-images.githubusercontent.com/81886982/234747753-a94f11ed-1939-49d5-a9af-8f9df96be20f.jpg)

**Comment by tarkah** [comment]
Checkout the tooltip widget! There's an example for it, too.

**Comment by Amazingkenneth** [comment]
Thanks!
